SKILL.tsv

```diff
@@ -4522,7 +4522,7 @@ SKILL_20150714_004521	$Increases the damage dealt on an enemy with [High Anchori
 SKILL_20150714_004522	$Increases the damage dealt on an enemy with [Critical Shot] by 1% per attribute level.
 SKILL_20150714_004523	$Increases the damage dealt on an enemy with [Time Bomb Arrow] 1% per attribute level.
 SKILL_20150714_004524	$Increases the damage dealt on an enemy with [Bounce Shot] by 1% per attribute level.
-SKILL_20150714_004525	$Knockbacks enemies hit by [Barrage] with a 10% chance per attribute level.
+SKILL_20150714_004525	$Knocked back enemies hit by [Barrage] with a 10% chance per attribute level.
```

Comment:
Most of the attribute description has "per attribute level" at the end of the description.

 So "Enemies hit by [Barrage] have a 10% chance of being knocked back per attribute level." fits better?